client/src/components/pages/dashboard/Dashboard.js

```diff
@@ -1,48 +1,48 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-import { connect } from "react-redux";
-import { logoutAdmin } from "../../../actions/authActions";
-import Button from "../../inputs/Button";
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { logoutAdmin } from '../../../actions/authActions';
```

Comment:
you always do paths like these, seems like there should be a better way

Reply:
I think there is but it seems very complicated. If we wanted to do this, I think it should be a new ticket, and since we don't have a lot of time, it might be better to just not do it.